### ISSUE TITLE
Add photo capture support

### ISSUE THREAD

**Issue by Darkwater23** [Darkwater23/MeadMate#10]

**Comment by gyles19** [comment]
This is the first feature I looked for: The ability to add photos to the batch data.

**Comment by Darkwater23** [comment]
Yes, I've had this on the list for a while. It's a bit tricky with how I built the app, but I think I can pull it off. I'll be starting another development session this week. I'll see what I can do.

**Comment by Darkwater23** [comment]
I experimented with adding a new activity to do this. It was ok, but it felt clunky. I'm going to see if I can get the webview to take a photo or select a photo from storage.

**Comment by Darkwater23** [comment]
I was able to start the camera and capture the image data in a separate sandbox project. I think I'm going to push the preference feature change first and come back to this in the next release. Almost there!
